app/src/main/java/com/example/dinhh/soundscape/data/remote/SoundscapeApi.kt

```diff
@@ -1,14 +1,41 @@
 package com.example.dinhh.soundscape.data.remote
 
 import com.example.dinhh.soundscape.data.entity.Token
+import io.reactivex.Observable
 import io.reactivex.Single
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.gson.GsonConverterFactory
 import retrofit2.http.Body
+import retrofit2.http.GET
 import retrofit2.http.POST
+import retrofit2.http.Query
 
 interface SoundscapeApi {
 
     @POST("api_auth/auth.php")
     fun login(@Body loginBody: LoginBody): Single<Token>
+
+
+    @GET("api_audio_search/index.php/?")
+    fun getSounds(@Query("key") key: String,
+                  @Query("category") category: String):
+            Observable<List<List<SoundNetworkModel.Response>>>
```

Comment:
I get a little bit confuse here the way you declare model. Should use Single instead of Observable anyway. Single is the same as Observable and the difference is that it just emits only one item